ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py

```diff
@@ -383,3 +395,32 @@ def test_numpy_nanargmin(
         axis=axis,
         keepdims=keep_dims,
     )
+
+    
+# extract
+@handle_frontend_test(
+    fn_tree="numpy.extract",
+    broadcastables=_broadcastable_duo(),
+)
+def test_numpy_extract(
```

Comment:
If you check out the intelligent-tests-1 error stack trace, you'll see some errors like
```
ivy/func_wrapper.py:191: in new_fn
    ret = fn(*args, **kwargs)
ivy/functional/backends/numpy/data_type.py:105: in astype
    return x.astype(dtype)
E   TypeError: Field elements must be 2- or 3-tuples, got ''int64''
```
Try re-generating the example which caused this to try and get to them bottom of the issue. This is done by adding the decorator printed in the error stack trace to the `test_numpy_extract` locally